crates/autopilot/src/decoded_settlement.rs

```diff
@@ -251,22 +248,12 @@ fn surplus(
     big_rational_to_u256(&normalized_surplus).ok()
 }
 
-fn fee(external_prices: &ExternalPrices, order: &Order) -> Option<U256> {
-    let solver_fee = u256_to_big_rational(&order.executed_solver_fee?);
-    tracing::trace!(?solver_fee, ?order.executed_solver_fee, "executed_solver_fee");
-
-    let fee = match order.kind {
-        model::order::OrderKind::Buy => {
-            solver_fee * u256_to_big_rational(&order.executed_amount)
-                / u256_to_big_rational(&order.buy_amount)
-        }
-        model::order::OrderKind::Sell => {
-            solver_fee * u256_to_big_rational(&order.executed_amount)
-                / u256_to_big_rational(&order.sell_amount)
-        }
-    };
```

Comment:
`solver_fee` does not have to be scaled anymore since the `OrderExecution` should already contain the scaled value for partial fills.

Reply:
Should this assumption be documented on the sovler_fee field in OrderExecution? I'm not sure, just saying.

Reply:
Are you sure about this?
Objective value computation still does the scaling here: https://github.com/cowprotocol/services/blob/186fd4f987af034eaef5c54b7b4adf24ef6d82e1/crates/solver/src/settlement.rs#L464